src/lib/components/ExperienceList.svelte

```diff
@@ -1,121 +1,76 @@
 <script lang="ts">
 	import ExperienceDisplay from '$lib/components/ExperienceDisplay.svelte';
-	import Modal from '$lib/components/Modal.svelte';
 	import Icon from '$lib/components/Icon.svelte';
-	import Input from '$lib/components/forms/Input.svelte';
-	import Button from '$lib/components/Button.svelte';
-	import { enhance } from '$app/forms';
-	import Checkbox from '$lib/components/forms/Checkbox.svelte';
+	import type { Experience } from '$lib/types/profile';
 
-	let displayEditExp = false;
-	let addNew = false;
-	let displayEndDate = false;
+	export let experiences: Experience[] | undefined;
+	export let openExperienceModal: null | ((isNew: boolean) => void) = null;
 
-	export let data;
-	export let action: boolean;
+	// avoid openExperienceModal possibly being undefined
+	function handleOpenModal(isNew: boolean) {
```

Comment:
You can write `() => openExperienceModal?.(isNew)` in the handler instead of redefining a function.